shadows/framework/src/main/java/org/robolectric/shadows/ShadowParcel.java

```diff
@@ -23,13 +26,48 @@
 import org.robolectric.annotation.RealObject;
 import org.robolectric.util.ReflectionHelpers;
 
+/**
```

Comment:
No javadoc needed here (we only want javadoc describing differences in the shadow's overall behavior).

Reply:
done